src/components/Global/Input/index.tsx

```diff
@@ -2,20 +2,24 @@ import React, { useState } from 'react';
 import styles from './Input.module.scss';
 
 type InputProps = {
-  type: 'text' | 'password';
+  type: 'text' | 'password' | 'email';
   id: string;
   required: boolean;
   placeholder?: string;
   maxLength?: number;
+  minLength?: number;
   labelText?: string;
+  setInputVal: React.Dispatch<React.SetStateAction<string>>;
```

Comment:
Is this type necessary? Does `() => void` not work?

Reply:
- The above way of defining the type is recommended for passing useState function as props though ( ) => void isn't entirely wrong. 
- setInputVal could either take in ( value: string | ( preVal: string) => { } ), and the second type in the return type is a dispatch. 
- Dispatch<SetStateAction<string>> is more generic; thus recommended.